backend/private-graph/graph/schema.graphqls

```diff
@@ -980,9 +980,9 @@ enum ReservedErrorObjectKey {
 	os
 	payload
 	request_id
+	secure_session_id
```

Comment:
I wanted to keep this consistent with logs and traces to not confuse the user with whether they should use `secure_session_id` or `session_secure_id`. This is not being used in the product, but wanted to confirm that this will not break anything in Grafana? c.c. @mayberryzane